src/commands/addBinding/settingSteps/AzureConnectionCreateStepBase.ts

```diff
@@ -33,7 +33,10 @@ export abstract class AzureConnectionCreateStepBase<T extends IBindingWizardCont
         progress.report({ message: localize('retrieving', 'Retrieving connection string...') });
 
         const result: IConnection = await this.getConnection(context);
-        let appSettingKey: string = `${result.name}_${nonNullProp(this._setting, 'resourceType').toUpperCase()}`;
+        let appSettingKey: string = context.useStorageEmulator ?
+            // if using the storage emulator, don't append the resource type to the app setting key
```

Comment:
nit: The comment explains what we're doing but not why. I think it's more helpful if the why was included.

Reply:
To be honest, I'm not really sure why @ejizba did this in the first place.

https://learn.microsoft.com/en-us/azure/azure-functions/functions-app-settings#azurewebjobsstorage

Here's the documentation about the setting `AzureWebJobsStorage` which is the default app setting linking to a storage account.

The old behavior would take the name of the azure resource and append the resource type (so like, naturins-storage_STORAGE) as the key. I'm wondering if `AzureWebJobsStorage` didn't exist at that time so he was just finding a common convention for the key?

Reply:
Not sure if he's talking about what I did or what you did @nturinski. The "_STORAGE" thing was just a random naming convention and I feel like I was probably just following an existing pattern from docs or the portal

In any case, I'm confused about what you're doing just like Alex probably is. Like, what does the emulator have to do with appending "_STORAGE"? Seems like both "AzureWebJobsStorage" and "randomkey_STORAGE" could use or not use the emulator, right?

Reply:
The issue is when I pass in `AzureWebJobsStorage` as the app key, it'll automatically append _STORAGE to the end of it.  You can see it in my screenshot where that happened.

Is it pretty common to have other storage connection strings outside of `AzureWebJobsStorage`?